theme/static/js/msrs/register_form.js

```diff
@@ -0,0 +1,25 @@
+$(document).ready(function(){
+    function loadCities() {
+      var selectedState = $('#id_state').val();
+      $.ajax({
+        url: '/get_cities_for_state/',
+        data: {'state': selectedState},
+        success: function(data) {
+          var citySelect = $('#id_city');
+          citySelect.empty();
+          $.each(data.cities, function(index, city) {
+            citySelect.append($('<option>', {
+              value: city,
+              text: city
```

Comment:
levando em conta city_label, city_value, aqui poderia ficar tipo assim:

```suggestion
              value: city_value,
              text: city_label
```